language/fluent/src/examples/builder/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from "./action";
```

Comment:
Should we be shipping examples of these components like this. IMO this should either just be: 
- In the docs
- An unpublished package
- Exported along side our existing reference assets from the player repo